src/main/views/courts/courts.njk

```diff
@@ -1,31 +1,73 @@
-{% extends "template.njk" %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
 
+{% extends "template.njk" %}
 
 {% block content %}
-  <h1 class="govuk-heading-xl">Courts and Tribunals</h1>
-
-  <table class="govuk-table" id="courts">
-    <thead class="govuk-table__head">
-      <tr class="govuk-table__row">
-        <th scope="col" class="govuk-table__header">Name</th>
-        <th scope="col" class="govuk-table__header">Last Updated</th>
-        <th scope="col" class="govuk-table__header"></th>
-        <th scope="col" class="govuk-table__header"></th>
-      </tr>
-    </thead>
-    <tbody class="govuk-table__body">
-    {% for court in courts | sort(attribute="name") %}
-      <tr class="govuk-table__row">
-        <td scope="row" class="govuk-table__cell">{{ court.name }}</td>
-        <td scope="row" class="govuk-table__cell">{{ court['updated_at'] }}</td>
-        <td scope="row" class="govuk-table__cell">
-          <a id="view-{{ court.slug }}" class="govuk-link" href="{{ factFrontendURL }}/courts/{{ court.slug }}">view</a>
-        </td>
-        <td scope="row" class="govuk-table__cell">
-          <a id="edit-{{ court.slug }}" class="govuk-link" href="/courts/{{ court.slug }}/edit#general">edit</a>
-        </td>
-      </tr>
-    {% endfor %}
-    </tbody>
-  </table>
+  <h1 class="govuk-heading-xl">Courts and tribunals</h1>
+
+  <div id = 'tableContainer'>
+
+    <input type='hidden' name='courtsHidden' id='courtsHidden' value='{{ courts | dump }}' />
```

Comment:
Would it be simpler to just display the table and then hide and/or sort the rows?